dashboard/db/migrate/20160519000515_drop_plc_enrollment_task_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class DropPlcEnrollmentTaskAssignments < ActiveRecord::Migration
+  def change
+    drop_table :plc_enrollment_task_assignments
```

Comment:
Cool.  I think it's reasonable to say that the reverse migration is just a no-op, but it's good to make it possible to rollback in case of emergency.